.docs/user-guide/storage-providers.md

```diff
@@ -656,14 +656,53 @@ libstorage:
           storagePoolName: storagePoolName
 ```
 
+## DigitalOcean
+Thanks to the efforts of our tremendous community, libStorage also has built-in
+support for DigitalOcean!
+
+<a class="headerlink hiddenanchor" name="digitalocean-block-storage"></a>
+<a class="headerlink hiddenanchor" name="dobs"></a>
+
+### DO Block Storage
+The DigitalOcean Block Storage (DOBS) driver registers a driver named `dobs`
+with the libStorage service registry and is used to attach and mount
+DigitalOcean block storage devices to DigitalOcean instances.
+
+!!! note
+    The DigitalOcean Block Storage driver currently only supports operating in
+    _local only_ mode where the libStorage server must be running on the same
+    host as the client.
+
+#### Requirements
+The DigitalOcean block storage driver has the following requirements:
+
+* Valid DigitalOcean account
```

Comment:
I'd add that you need an access token, and here is the link/doc on how to generate one: https://www.digitalocean.com/community/tutorials/how-to-use-the-digitalocean-api-v2#how-to-generate-a-personal-access-token